doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -169,3 +169,44 @@ Currently, the following intents are defined within the ``IntentEnum`` enumerato
   * ``kEndUserSupport`` - Logs created by the product maker to be used for end-user support.
   * ``kNetworkDiag``- Logs created by the network layer to be used for network diagnostic.
   * ``kCrashLogs`` - Logs created during a device crash, to be obtained from the node.
+
+Crash logs
+----------
+
+The crash logs module is a part of the diagnostic log module and contains the data of the recent device crash.
+When a crash occurs the device saves the crash data to the defined retained RAM partition and therefore to enable crash log support the :ref:`ug_matter_diagnostic_logs_snippet` must be added to the build.
+Only the last crash data will be stored in the device's memory, and because of that if another crash occurs the previous one will be overwritten.
+After receiving the read request from the Matter controller, the device reads out the crash data from the retained memory or settings storage and creates human readable logs at runtime.
+Finally, the device sends converted logs to the Matter controller as a response.
+
+The :ref:`ug_matter_diagnostic_logs_snippet` enables support for copying the crash data to the Zephyr's settings storage at the first boot after a crash by default.
+This operation allows the device to keep crash data even if the power breakdown occurs however it will use some space within the settings storage partition and should be taken into account while setting the partition size.
```

Comment:
```suggestion
This operation allows the device to keep crash data even if a power breakdown occurs, but it will use some space within the settings storage partition and you need to take it into account while setting the partition size.
```